TriforkSwiftExtensions/TriforkSwiftExtensions/Numeric/NumericExtension.swift

```diff
@@ -30,6 +29,6 @@ public extension Double {
         let formatter = NumberFormatter()
         formatter.maximumSignificantDigits = digits
         formatter.minimumSignificantDigits = digits
-        return formatter.string(from: NSNumber(value: self))
+        return formatter.string(from: self as! NSNumber)
```

Comment:
This should always be safe as Numeric == NSNumber. 
I can safely cast and return nil if you prefer that @tkctrifork

Reply:
I see why it is safe, since the `NSNumber` constructor interface supports all types of `Numeric` implementations, but this code might be unsafe if Apple decides to change this. The `NSNumber` doesn't support the `Numeric` protocol, which is why I think we should unwrap this safely before passing it to the constructor.

Reply:
I have made a safe cast.